lasagne/layers/recurrent.py

```diff
@@ -222,6 +222,23 @@ def __init__(self, incoming, input_to_hidden, hidden_to_hidden,
 
         super(CustomRecurrentLayer, self).__init__(incomings, **kwargs)
 
+        input_to_hidden_layers = helper.get_all_layers(input_to_hidden)
+        input_to_hidden_in_lyrs = [lyr for lyr in input_to_hidden_layers
+                                   if isinstance(lyr, InputLayer)]
```

Comment:
Can you combine this into one call? The fewer local variables are initialized, the less I need to worry about when reading the code. I'd also prefer not to use abbreviations:
```python
input_to_hidden_input_layers = [layer for layer
                                in helper.get_all_layers(input_to_hidden)
                                if isinstance(layer, InputLayer)]
```